zerver/tests/test_events.py

```diff
@@ -2609,6 +2610,35 @@ def test_notification_setting_event_not_sent(self) -> None:
         )
         check_user_settings_update("events[0]", events[0])
 
+    def test_admin_can_edit_multiuse_invite(self) -> None:
+        """
+        This is a sanity check test that occurs in the best-case scenario,
+        taken from the two tests from `test_invite.multiUseInviteTest` below.
+        * `test_multiuse_invite_edits_with_malformed_parameters`
+        * `test_non_previleged_multiuse_invite_edits`
+        """
+        self.user_profile = self.example_user("iago")
+        self.login("iago")
+
+        initial_invited_as = PreregistrationUser.INVITE_AS["REALM_ADMIN"]
+        initial_streams = [get_stream("Denmark", self.user_profile.realm)]
+        invite_expires_in_minutes = 2 * 24 * 60
+        do_create_multiuse_invite_link(
+            self.user_profile,
+            initial_invited_as,
+            invite_expires_in_minutes,
+            initial_streams,
+        )
+
+        initial_multiuse_object = MultiuseInvite.objects.get()
+        self.verify_action(
+            lambda: do_edit_multiuse_invite_link(
+                initial_multiuse_object, initial_invited_as, initial_streams
+            ),
+            state_change_expected=False,
+            event_types=["invites_changed"],
+        )
```

Comment:
You should add a call to `check_invites_changed` to verify the contents of the event.

Reply:
```py
self.verify_action(
    lambda: do_edit_multiuse_invite_link(
        initial_multiuse_object, initial_invited_as, initial_streams
    ),
    state_change_expected=False,
    event_types=["invites_changed"],
)
```

I inferred that passing `event_types=[...]` to `verify_action` checked the event type for me. Now that I see the test file, the helper `check_invites_changed` is used consistently.

I want to know if consistency is the reason for this review comment or was my inference very off. Please lmk.

Reply:
`event_types` is useful when we want to test only events of a particular type where multiple type events are sent. To check the event object, which includes type field, we use the `check_..` functions.